sechub-commons-model/src/test/resources/sechub_license_scan_wrong_source_config_example.json

```diff
@@ -0,0 +1,14 @@
+{
+  "apiVersion" : "1.0",
+  "data" : {
+    "source" : [ {
```

Comment:
While testing I used "source" instead of "sources" it took me quite some time to understand where the issue is. I wrote the test to check what happens if I just write "source" instead of "sources". 

The question would be should we give a hint in the error message? For example, by writing: `"source" is not acceptable. Do you mean "sources"`.

Reply:
I will remove this test and instead open an issue for the client. The client can analyze the user provided configuration and give some hints.